sdk/communication/AzureCommunicationCommon/Source/Identifiers.swift

```diff
@@ -52,14 +52,11 @@ import os.log
     var kind: IdentifierKind { get }
 }
 
-internal enum Prefix {
+enum Prefix {
     public static let PhoneNumber = "4:"
-    public static let Bot = "28:"
-    public static let BotPublicCloud = "28:orgid:"
-    public static let BotDodCloud = "28:dod:"
-    public static let BotDodCloudGlobal = "28:dod-global:"
-    public static let BotGcchCloud = "28:gcch:"
-    public static let BotGcchCloudGlobal = "28:gcch-global:"
+    public static let TeamsAppPublicCloud = "28:orgid:"
+    public static let TeamsAppDodCloud = "28:dod:"
+    public static let TeamsAppGcchCloud = "28:gcch:"
     public static let TeamUserAnonymous = "8:teamsvisitor:"
```

Comment:
```suggestion
    public static let TeamsUserAnonymous = "8:teamsvisitor:"
```
let's fix all old typo in this enum

Reply:
Typo fixed.